communication/include/communication/CommDefs.hpp

```diff
@@ -48,14 +48,14 @@ struct BaseCommConfig {
   NodeNum selfId;
 
   BaseCommConfig(CommType type,
-                 std::string host,
+                 const std::string &host,
```

Comment:
Why CommDefs.hpp changes are part of this PR?

Reply:
Tidy checks yield errors about this